activestorage/test/models/attached/one_test.rb

```diff
@@ -16,6 +16,22 @@ class ActiveStorage::OneAttachedTest < ActiveSupport::TestCase
     ActiveStorage::Blob.all.each(&:delete)
   end
 
+  test "creating an user with a File as avatar" do
```

Comment:
Tiny nitpick, the first sound of user is not a vowel so it should be:
```suggestion
  test "creating a user with a File as avatar" do
```
https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

Reply:
As the tests shouldn't care about the type of record or the attachment name, what do you think about:
```suggestion
  test "creating a record with a File as attached" do
```